sofa-tracer-plugins/sofa-tracer-zipkin-plugin/src/main/java/com/alipay/sofa/tracer/plugins/zipkin/ZipkinSofaTracerSpanRemoteReporter.java

```diff
@@ -52,7 +52,7 @@ public ZipkinSofaTracerSpanRemoteReporter(RestTemplate restTemplate, String base
 
     @Override
     public void onSpanReport(SofaTracerSpan span) {
-        if (span == null) {
+        if (span == null || !span.getSofaTracerSpanContext().isSampled()) {
```

Comment:
Does `span.getSofaTracerSpanContext() ` need check NPE ?

Reply:
> Does `span.getSofaTracerSpanContext() ` need check NPE ?

Don't need. When we are building a Span, the SpanContext must not be null.